script.js

```diff
@@ -0,0 +1,14 @@
+const productList = document.getElementById("product-list");
+const selectedProductDetailsDiv = document.getElementById("selected-product-details");
+
+productList.addEventListener("click", function(e) {
```

Comment:
For simplifying it is always better to add a function reference here, and then define function outside. For Ex:
```
function addProductHandler(){
 // function code
}

productList.addEventListener('click', addProductHandler);
```

Reply:
Thanks, @akshayraichur ! Moved addProductHandler outside the event listener as suggested for better readability and reuse. Let me know if there's anything else!